scripts/execute_maestro_tests.sh

```diff
@@ -3,6 +3,26 @@ set -o pipefail
 set -x
 set -e
 
+# Maestro tags to be executed. see https://maestro.mobile.dev/cli/tags
+MAESTRO_TAGS=""
```

Comment:
Allows the maestro script to run tests by tag (instead of executing the whole test suite)